doc: peripheral_hids_keyboard,mouse encryption notice (#6454)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_hids_keyboard/README.rst

LED 4:
   Indicates if an NFC field is present.

Configuration
*************

|config|

Setup
=====

> :kconfig:`BT_HIDS_DEFAULT_PERM_RW_ENCRYPT` is not required by HID GATT specification, but certain systems do not allow connections to unencrypted HID devices.

Reviewer: ```suggestion
The HID GATT specification does not require encryption (:kconfig:`CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT`), but some systems do not allow connections to unencrypted HID devices.
```

> The HID GATT specification does not require encryption (:kconfig:`CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT`), but some systems do not allow connections to unencrypted HID devices.

Reviewer 2: Again I would slightly change it:
```suggestion
The HID service specification does not require encryption (:kconfig:`CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT`), but some systems disconnect from the HID devices which do not support security.
```

> The HID service specification does not require encryption (:kconfig:`CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT`), but some systems disconnect from the HID devices which do not support security.

Reviewer: ```suggestion
The HID service specification does not require encryption (:kconfig:`CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT`), but some systems disconnect from the HID devices that do not support security.
```
